rootfs_overlay/etc/unbound/unbound.conf

```diff
@@ -83,6 +86,15 @@ server:
     # (requires CAP_NET_ADMIN or privileged)
     # so-rcvbuf: 1m
 
+    # Increase cache size to utilize more RAM
+    msg-cache-size: 128m
+    rrset-cache-size: 256m
```

Comment:
Is this going to be an issue on smaller devices like Raspberry Pi Zero?

Reply:
Good hint. I halve the size, that should not be a problem. Default is 4m/4m.